src/timeline/TimelineModel.cpp

```diff
@@ -375,6 +375,15 @@ TimelineModel::TimelineModel(TimelineViewManager *manager, QString room_id, QObj
         connect(&events, &EventStore::updateFlowEventId, this, [this](std::string event_id) {
                 this->updateFlowEventId(event_id);
         });
+        connect(&events,
+                &EventStore::messageSent,
+                this,
+                [this](std::string txn_id, std::string event_id) {
+                        if (edit_.toStdString() == txn_id) {
```

Comment:
can you do that for reply as well?

Reply:
Oh, right, I wanted to but forgot. There doesn't appear to be a corresponding signal for `replyChanged`, I hope I don't need to emit anything.